travel.py

```diff
@@ -0,0 +1,30 @@
+#Brute force solution to solve the problem
+#Time compelxity: O(3n)
```

Comment:
Time time complexity would be supposedly more because of the itertools.permutations call.